_pytest/fixtures.py

```diff
@@ -1068,6 +1068,9 @@ def parsefactories(self, node_or_obj, nodeid=NOTSET, unittest=False):
         self._holderobjseen.add(holderobj)
         autousenames = []
         for name in dir(holderobj):
+            # Skip properties before the following getattr executes them.
+            if isinstance(getattr(type(holderobj), name, None), property):
```

Comment:
hmm, maybe this should use `safe_getattr` from `_pytest.compat`? Probably a getattr on a type will never raise something, but you never know with metaclasses and stuff I guess 😆